fractalis/config.py

```diff
@@ -35,4 +35,12 @@
 # Whether to verify the certs of https data sources
 ETL_VERIFY_SSL_CERT = False
 
+# Authorization
+# Disable the authorization based on the user access token. Not recommended!
+AUTHORIZATION_DISABLED = False
+# OIDC refresh token enabling an offline access
+OIDC_OFFLINE_TOKEN = '<your_offline_token>'
```

Comment:
One comment, these are default values... Shouldn't these be `None`, to make sure they are always overwritten (and fail somewhere when no value has been set?)

Reply:
good point!